Multisig/UI/Dapps/DappsDataSource.swift

```diff
@@ -21,11 +21,20 @@ class DappsDataSource {
     let dapps: [DappData]
 
     init() {
```

Comment:
needs checking manually if not crashing when the WalletConnect experimental is enabled

Reply:
done